superchain/extra/genesis-system-configs/sepolia-dev-0/oplabs-devnet-0.json

```diff
@@ -0,0 +1,6 @@
+{
+  "batcherAddr": "0x19cc7073150d9f5888f09e0e9016d2a39667df14",
+  "overhead": "0x00000000000000000000000000000000000000000000000000000000000000bc",
+  "scalar": "0x00000000000000000000000000000000000000000000000000000000000a6fe0",
```

Comment:
Different from Base, but shouldn't be a problem, and matches the rollup-config contents when I query the sequencer RPC.

Reply:
Agree, and we'll both change to the same values after Ecotone activation.